docs/GetStarted/getting_started_linux_cmake.md

```diff
@@ -50,6 +50,14 @@ We recommend Clang. GCC is not fully supported.
 $ sudo apt install clang
 ```
 
+### Install Python
```

Comment:
Probably should specify "Python 3" here, since people are still for some reason using Python 2...

Reply:
That's fair. I wanted to avoid specifying out of the principle that it really _shouldn't_ be ambiguous anymore.